cmd/vic-init/main_linux.go

```diff
@@ -201,3 +254,28 @@ func defaultIP() string {
 
 	return toolbox.DefaultIP()
 }
+
+func startSignalHandler() {
```

Comment:
Do we have a pattern for sharing base functionality between `vic-init` and `tether? Consistency in the way we handle signals seems like something we'd want to preserve in the future (if for no other reason than developer sanity), and duplicating the code allows for unintentional drift.

Reply:
There is a good chunk of shared function in `lib/tether`, however the code in `cmd/tether` and `cmd/vic-init` is expressly that needed to support different behaviours.

What you're proposing is definitely desirable but I think necessitates the toolbox interface for system interaction that I noted above. This would be needed to order to avoid the package local `tthr` skyhook that is used in this function.

I'll revisit this; the most that I'm likely to do prior to designing the system interaction portion of vcsim is to make this a `lib/tether` function that takes `tthr` as an argument.